Description: Switch back to the JNI based snappy-java implementation (the pure Java version isn't in Debian yet)
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/src/main/java/org/codehaus/plexus/archiver/snappy/SnappyCompressor.java
+++ b/src/main/java/org/codehaus/plexus/archiver/snappy/SnappyCompressor.java
@@ -19,7 +19,7 @@
 import java.io.IOException;
 import org.codehaus.plexus.archiver.ArchiverException;
 import org.codehaus.plexus.archiver.util.Compressor;
-import org.iq80.snappy.SnappyFramedOutputStream;
+import org.xerial.snappy.SnappyFramedOutputStream;
 import static org.codehaus.plexus.archiver.util.Streams.bufferedOutputStream;
 import static org.codehaus.plexus.archiver.util.Streams.fileOutputStream;
 
--- a/src/main/java/org/codehaus/plexus/archiver/snappy/SnappyUnArchiver.java
+++ b/src/main/java/org/codehaus/plexus/archiver/snappy/SnappyUnArchiver.java
@@ -24,7 +24,7 @@
 
 import org.codehaus.plexus.archiver.AbstractUnArchiver;
 import org.codehaus.plexus.archiver.ArchiverException;
-import org.iq80.snappy.SnappyFramedInputStream;
+import org.xerial.snappy.SnappyFramedInputStream;
 import static org.codehaus.plexus.archiver.util.Streams.bufferedInputStream;
 import static org.codehaus.plexus.archiver.util.Streams.bufferedOutputStream;
 import static org.codehaus.plexus.archiver.util.Streams.copyFully;
--- a/src/main/java/org/codehaus/plexus/archiver/tar/TarArchiver.java
+++ b/src/main/java/org/codehaus/plexus/archiver/tar/TarArchiver.java
@@ -44,7 +44,7 @@
 import org.codehaus.plexus.components.io.resources.PlexusIoResource;
 import org.codehaus.plexus.util.IOUtil;
 import org.codehaus.plexus.util.StringUtils;
-import org.iq80.snappy.SnappyOutputStream;
+import org.xerial.snappy.SnappyOutputStream;
 
 /**
  * @author <a href="mailto:evenisse@codehaus.org">Emmanuel Venisse</a>
--- a/src/main/java/org/codehaus/plexus/archiver/tar/TarUnArchiver.java
+++ b/src/main/java/org/codehaus/plexus/archiver/tar/TarUnArchiver.java
@@ -35,7 +35,7 @@
 import org.codehaus.plexus.archiver.ArchiverException;
 import org.codehaus.plexus.archiver.util.Streams;
 import org.codehaus.plexus.components.io.filemappers.FileMapper;
-import org.iq80.snappy.SnappyFramedInputStream;
+import org.xerial.snappy.SnappyFramedInputStream;
 
 /**
  * @author <a href="mailto:evenisse@codehaus.org">Emmanuel Venisse</a>
@@ -154,7 +154,7 @@
         }
         else if ( compression == UntarCompressionMethod.SNAPPY )
         {
-            return new SnappyFramedInputStream( istream, true );
+            return new SnappyFramedInputStream( istream );
         }
         else if ( compression == UntarCompressionMethod.XZ )
         {
